Makefile

```diff
@@ -1,30 +1,32 @@
+REBAR = rebar3
```

Comment:
I removed the `rebar` binary and added the dependency on [quintana](https://github.com/puzza007/quintana), which was missing. I also bumped the version number.